src/fetchtastic/downloader.py

```
@@ -1442,23 +1674,26 @@ def _send_ntfy_notification(
         pass
 
 
-def _get_latest_releases_data(url: str, scan_count: int = 10) -> List[Dict[str, Any]]:
+def _get_latest_releases_data(
+    url: str, scan_count: int = 10, github_token: Optional[str] = None
+) -> List[Dict[str, Any]]:
     """
-    Return a list of the most recent releases fetched from a GitHub releases API endpoint.
+    Return a list of most recent releases fetched from a GitHub releases API endpoint.
 
     Fetches up to `scan_count` releases from the provided GitHub API `url` (clamped to 1–100),
```

Ruff (0.14.1)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)